src/main/java/uk/gov/hmcts/ccd/domain/model/casedataaccesscontrol/AccessProcess.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol;
+
+public enum AccessProcess {
```

Comment:
not sure enums in our case are a good idea. When they introduce additional values our deserialisation process might break. Also there's more code to update when new values are introduced.
The thing is we don't really need to use these values in our domain that much, all we need to do is doing String equality checks in few places. Therefore I think we can apply YAGNI and I would leave them as String for now

Reply:
code is not going to break, not using valueOf(name) in this case